### DISCUSSION TITLE
Is it possible to use one config file for all repositories?

### DISCUSSION THREAD

**Discussion by rlnt** [repository-settings/app#526] [Q&A]
Hello!
I'd like to know if it was possible to only use one configuration file for all repositories.
My organisation mostly focuses on Minecraft mods and packs so the labels and tags can be the same on all repositories.
I have set up my settings config in the `.github` repository of the organization. Can I use this config for all other repositories as well?

**Comment by travi** [comment]
extension in each repository is supported through [inheritance](https://github.com/probot/settings#inheritance)

**Reply by dgokcin** [reply]
@travi in this case. what happens if you change a setting in the base `settings.yml` file? would updating a field in the base file trigger the same thing in the child repos?

**Reply by erazemk** [reply]
@travi The link you posted at the time doesn't seem to point to anything anymore, and the docs don't seem to mention how to set up inheritance. How can this be currently done?

**Reply by travi** [reply]
https://github.com/repository-settings/app/blob/master/docs/configuration.md#inheritance would be the updated location for those docs

**Reply by erazemk** [reply]
Yes, I read that, but the instructions are very short and unclear, what is the base repo, how can I define it?

**Reply by dgokcin** [reply]
@erazemk you might wanna give the comments under this issue a shot. Also reading what other people has written about inheritance could also be helpful.

- https://github.com/repository-settings/app/issues/391#issuecomment-797795120
- https://github.com/repository-settings/app/issues?q=is%3Aissue+is%3Aopen+inherit

**Reply by erazemk** [reply]
Thank you, will check them out
